heron/statemgrs/tests/python/zkstatemanager_unittest.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''ZkStateManager unittest'''
+import unittest2 as unittest
+
+from heron.statemgrs.src.python.zkstatemanager import ZkStateManager
+
+
+class ZkStateManagerTest(unittest.TestCase):
+  """Unittest for ZkStateManager"""
+
+  class NoseFlute:
```

Comment:
This took me a little while to figure out the NoseFlute naming, I'm a little slow. :) Can we call this MockKazoo or something more boring, but more informative?